omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/RegisterPatientFragmentController.java

```diff
@@ -6,21 +6,8 @@
 import org.codehaus.jackson.JsonNode;
 import org.codehaus.jackson.node.ArrayNode;
 import org.joda.time.DateTimeComparator;
-import org.openmrs.Concept;
-import org.openmrs.Encounter;
-import org.openmrs.EncounterRole;
-import org.openmrs.EncounterType;
-import org.openmrs.Obs;
-import org.openmrs.Patient;
-import org.openmrs.PersonAddress;
-import org.openmrs.PersonAttribute;
-import org.openmrs.PersonName;
-import org.openmrs.api.ConceptService;
-import org.openmrs.api.DuplicateIdentifierException;
-import org.openmrs.api.EncounterService;
-import org.openmrs.api.InvalidCheckDigitException;
-import org.openmrs.api.ObsService;
-import org.openmrs.api.PatientIdentifierException;
+import org.openmrs.*;
```

Comment:
IntelliJ IDEA might have optimized imports. Does this mean I've to generate a PR again?